homeassistant/components/binary_sensor/mychevy.py

```diff
@@ -0,0 +1,68 @@
+"""Support for MyChevy sensors."""
```

Comment:
Please update the module docstring to follow the regular format with a link to the docs:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/luftdaten.py#L1-L6